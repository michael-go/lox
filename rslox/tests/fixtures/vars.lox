var beverage = "cafe au lait";
beverage = "espresso";
var breakfast = "beignets with " + beverage;
print breakfast;

no_such_var = 42;

breakfast + "foo" = "bar";