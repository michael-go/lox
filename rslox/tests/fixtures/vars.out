# exit code: 65
# stdout:
"beignets with espresso"

# stderr:
[line 8] Error at '=': Invalid assignment target
Runtime error: Undefined variable no_such_var
[line 6] in <script>

